### PR TITLE
feat: Implement dashboards, add majority of application configuration

### PR DESCRIPTION
**Pull request by danielptv**
[ODDSK-71](https://h-ka-team-rdqzrlfpomci.atlassian.net/browse/ODDSK-71), [ODDSK-218](https://h-ka-team-rdqzrlfpomci.atlassian.net/browse/ODDSK-218), [ODDSK-217](https://h-ka-team-rdqzrlfpomci.atlassian.net/browse/ODDSK-217).

* Implement dashboards configuration format
* Implement configurable dashboards
* Configure homepage and dashboards based on provided use cases
* Multiple fixes to make GeoMap and BarChart resize responsively
* Fix conversion of name/id to path (handling of special characters)
* Fix BarChart crashing when filter returns no results
* Fix maps overlapping search results (wrong zIndex)
* Fix table overflowing when there are a lot of columns
* Fix: Escape unescaped dots in the CSV header to ensure correct parsing
* Use licensed image on the homepage
* Cache configuration and transformed resources to improve performance

Homepage: https://deploy-preview-165--open-data-dashboard.netlify.app/de/home
Dashboard "Bevölkerung": https://deploy-preview-165--open-data-dashboard.netlify.app/en/dashboard/bev%C3%B6lkerung-1
Dashboard "Verkehr": https://deploy-preview-165--open-data-dashboard.netlify.app/de/dashboard/verkehr-und-infrastruktur-2
Dashboard "Kultur": https://deploy-preview-165--open-data-dashboard.netlify.app/de/dashboard/kultur-und-%C3%B6ffentliches-leben-3
Dashboard "Für Studierende": https://deploy-preview-165--open-data-dashboard.netlify.app/en/dashboard/f%C3%BCr-studierende-5

### CODE REVIEW

**Comment by danielptv** [comment]
@Zerdados I have verified the configurations added by @ChromatisGit. Double-check only if you feel like it  :=)

**Review by Zerdados — COMMENTED** [review]
Looks great overall, just a few minor comments

File: config/dashboards.app.config.yml [hunk 1]
```diff
@@ -0,0 +1,227 @@
...
+            - - type: LINK_EXTERNAL
+                text: Stau.info
+                icon: car-front-fill
+                target: https://stau.info/stadt/karlsruhe
+                backgroundColor: "#004F8F"
+              - type: RESOURCE
+                resourceId: 528cc3da-baf5-403d-a6c7-906113281c83
+    - id: "3"
+      name: Kultur und öffentliches Leben
+      description: Neben Kulturangeboten und Wochenmärkte, findest Du hier Informationen zu städtischen Behörden.
```
[the remark this answers is not in the document]
Comment (Zerdados) [reply config/dashboards.app.config.yml]: The comma is still there

File: config/pdfs.app.config.yml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    - id: c867155b-5f28-4d25-882e-422fb3ae18aa
+      name: Mietspiegelrechner für Wohnungen 2023/24
+      description: |
+          Der Mietspiegel ist eine Übersicht über die Mieten, die in Karlsruhe für frei finanzierte Wohnungen bezahlt werden (ortsübliche Vergleichsmiete).
+      source: https://web6.karlsruhe.de/Stadtentwicklung/statistik/mietspiegelrechner/mietspiegel-2023.html
+      type: HTML
+    - id: a73cda70-e38f-4f6b-9d7a-d1a004f8cc0d
+      name: Mietspiegelrechner für Einfamilienhäuser 2023/24
+      description: |
+          Der Mietspiegel ist eine Übersicht über die Mieten, die in Karlsruhe für frei finanzierte Wohnungen bezahlt werden (ortsübliche Vergleichsmiete).
```
[the remark this answers is not in the document]
Comment (Zerdados) [reply config/pdfs.app.config.yml]: You used the same text as for the other Mietspiegelrechner, however, this one is for "Einfamilienhäuser". Please change the text to the one in my original comment

**Review by Zerdados — COMMENTED** [review]
Only 2 comments left

**Review by Zerdados — APPROVED** [review]

**Pull request merged**